src/tools/genpolicy/rules.rego

```diff
@@ -274,18 +274,9 @@ allow_by_sandbox_name(p_oci, i_oci, p_storages, i_storages, s_name) {
     print("allow_by_sandbox_name: true")
 }
 
-allow_sandbox_name(p_s_name, i_s_name) {
-    print("allow_sandbox_name 1: start")
-
-    p_s_name == i_s_name
-
-    print("allow_sandbox_name 1: true")
-}
 allow_sandbox_name(p_s_name, i_s_name) {
     print("allow_sandbox_name 2: start")
```

Comment:
```suggestion
    print("allow_sandbox_name: start")
```
Nit: there's only one `allow_sandbox_name` now. Also at the end of the function.

Reply:
Updated. Thanks!